app/config/message.js

```diff
@@ -0,0 +1,52 @@
+import Joi from 'joi'
+import environments from '../constants/environments.js'
+
+const schema = Joi.object({
+  messageQueue: {
+    host: Joi.string(),
+    username: Joi.string(),
+    password: Joi.string(),
+    useCredentialChain: Joi.bool().default(false),
+    managedIdentityClientId: Joi.string().optional(),
+    appInsights: Joi.object()
+  },
+  receiverSubscription: {
+    address: Joi.string(),
+    topic: Joi.string(),
+    type: Joi.string().allow('subscription')
+  }
+})
+
```

Comment:
Will start looking into the Convict docs. Is this because we've moved to using ESM instead of CommonJS?

Reply:
No convict is just better for managing and validating config.  The FCP template was changed to use it instead of this joi approach.

Joi just handles validation so the rest is up to you which invites inconsistency.

Reply:
Gotcha, will look at the template and use that as an example for how to use Convict. 👍🏽

Reply:
Have set up the Service Bus (i.e. message) config to now use Convict instead of Joi.